manifests/a/AntoniKepinski/ParsifyDesktop/2.0.0-rc.1/AntoniKepinski.ParsifyDesktop.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AntoniKepinski.ParsifyDesktop
+PackageVersion: 2.0.0-rc.1
+PackageLocale: en-US
+Publisher: Antoni Kępiński
```

```suggestion
Publisher: Parsify Technologies
```
This must match control panel